app/models/manageiq/providers/ibm_power_vc/inventory/parser/cloud_manager.rb

```diff
@@ -1,2 +1,210 @@
 class ManageIQ::Providers::IbmPowerVc::Inventory::Parser::CloudManager < ManageIQ::Providers::Openstack::Inventory::Parser::CloudManager
+  def parse_vm(vm, hosts)
+    super
+    server = persister.vms.find_or_build(vm.id.to_s)
+
+    # saving processor type and amount
+    flavor = get_flavor(vm)
+    processors = flavor.try(:vcpus)
+    persister.vms_and_templates_advanced_settings.build(
+      :resource     => server,
+      :display_name => _('Entitled Processors'),
+      :name         => 'entitled_processors',
+      :description  => _('The number of entitled processors assigned to the VM'),
+      :value        => processors,
+      :read_only    => true
+    )
+
+    # saving pin_policy
+    pin_policy = calculate_pinning(vm)
+    persister.vms_and_templates_advanced_settings.build(
+      :resource     => server,
+      :display_name => _('Pin Policy'),
+      :name         => 'pin_policy',
+      :description  => _('VM pinning policy to use [none, soft, hard]'),
+      :value        => pin_policy,
+      :read_only    => true
+    )
+
+    persister.vms_and_templates_advanced_settings.build(
+      :resource     => server,
+      :display_name => _('Physical Page Table (PPT) ratio'),
+      :name         => 'ppt_ratio',
+      :description  => _('The PPT ratio is the ratio of the maximum memory to the size of the Physical Page Table.'),
+      :value        => flavor.metadata["powervm:ppt_ratio"] || "",
+      :read_only    => true
+    )
+
+    persister.vms_and_templates_advanced_settings.build(
+      :resource     => server,
+      :display_name => _('Enforce Affinity Check'),
+      :name         => 'enforce_affinity_check',
+      :description  => _('Enforce the affinity score for the partition is greater than the destination host'),
+      :value        => flavor.metadata["powervm:enforce_affinity_check"] || "false",
+      :read_only    => true
+    )
+
+    persister.vms_and_templates_advanced_settings.build(
```

Comment:
Let's reconcile this between entitled processors (above) -- i.e., we don't need to show both because entitled processors and processor units are the same logical entity - and for purposes of this PR, we should only show the one that's in effect for the VM in PowerVC.

I don't recall which attribute to use off-hand, but looking at the PowerVC GUI and looking at the metadata in the API response will tell you which value to use.